### ISSUE TITLE
Declare twisted.web2 stable and supported, de-emphasize twisted.web

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2085]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#2085|
|Type|enhancement|
|Created|2006-09-13 00:58:22Z|

There can be only one.

Twisted's web situation has been fragmentary and confusing for too long.  twisted.web is still being actively maintained in parallel with twisted.web2 and this is a strain on our exceedingly limited resources.  This ticket formalizes the long shared but unstated goal of unifying the entire Twisted web development sphere and recommending the One True Path to people building HTTP-enabled or web-based applications.

The goal of this ticket is to produce a web2 "golden master" - the first version that we ship to our publisher, and we can distribute patches on the game's website after the fact.

Erm, I mean... the goal of this ticket is to produce the first version whose API is stable and supported, and whose implementation is good enough to use.

This depends on all tickets marked in the [Web2-Gold-Master](http://twistedmatrix.com/trac/query?status=new&status=assigned&status=reopened&milestone=Web2-Gold-Master) milestone. [Divmod ticket 1066, the Nevow port to web2](http://divmod.org/trac/ticket/1066).

This ticket does _not_ mean that web2 is finished, or even bug free, but that it is:

 * fast enough for real production work - twisted.web sets a good baseline here
 * has stable, supported core APIs that members of the core Twisted team are all satisfied with
 * supported by de-facto community Twisted Web standards, such as Nevow
 * still supporting existing users, such as Zope3
 * "robust enough" - no clearly visible bugs should be noticed during normal use, no show stoppers even under sophisticated configurations and high load

<details><summary>Searchable metadata</summary>

```
trac-id__2085 2085
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__Web2_Gold_Master Web2-Gold-Master
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__web2 web2
keywords__ 
time__1158109102000000 1158109102000000
changetime__1213697302000000 1213697302000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

Since development effort has shifted to porting web2 features back to web, I don't think this ticket is valid any longer.  We should probably do something about the `web2-gold-master` milestone, too...

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
